src/OSPSuite.UI/Controls/UxMasterDetailGridView.cs

```diff
@@ -0,0 +1,13 @@
+using DevExpress.XtraGrid;
+
+namespace OSPSuite.UI.Controls
+{
+   public class UxMasterDetailGridView : UxGridView
+   {
+      protected override string ViewName => "UxMasterDetailGridView";
```

Comment:
When I tried to apply this to UxGridView some exceptions were thrown from DevExpress. I created this class so I could isolate the behavior to just where we need these clones to be made from a subclass.

We need the grid control from the MultiplePIResultsView to create this kind of grid view when it makes a clone of optimizedParameterView.